src/overloads/arrays.jl

```diff
@@ -187,6 +187,22 @@ function LinearAlgebra.logabsdet(A::AbstractMatrix{D}) where {D<:Dual}
     t1, t2 = LinearAlgebra.logabsdet(tracers)
     return (D(p1, t1), D(p2, t2))
 end
+function LinearAlgebra.:\(A::AbstractMatrix{D}, B::AbstractVector) where {D<:Dual}
+    primals, tracers = split_dual_array(A)
+    p = primals \ B
+    t = tracers \ B
+    return D.(p, t)
+end
+function LinearAlgebra.:\(A::AbstractMatrix, B::AbstractVector{D}) where {D<:Dual}
+    return D.(A) \ B
+end
+function LinearAlgebra.:\(A::AbstractMatrix{D}, B::AbstractVector{D}) where {D<:Dual}
+    A_primals, A_tracers = split_dual_array(A)
+    B_primals, B_tracers = split_dual_array(B)
+    p = A_primals \ B_primals
+    t = A_tracers \ B_tracers
+    return D.(p, t)
```

Comment:
These should probably also say `Dual` instead of `D`, as the primal type might be promoted.